src/main/java/org/numenta/nupic/network/Layer.java

```diff
@@ -1832,7 +1850,7 @@ public ManualInput call(ManualInput t1) {
                         
                         CLAClassifier c = (CLAClassifier)t1.getClassifiers().get(key);
                         ClassifierResult<Object> result = c.compute(
-                            recordNum, inputMap, t1.getSDR(), true, true);
+                            recordNum, inputMap, t1.getSDR(), isLearn, true);
```

Comment:
I like this! Question @cogmission, would it be useful to get rid of `infer` hardcoding like you did with `isLearn` ?